content/posts/networkx/outreachy2023/contribution-phase/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Outreachy internships: My experience as a first-time contributor in Open-Source"
+date: 2023-02-26T22:15:58-03:00
+draft: false
+description: "
+Some tips to help you during the contribution phase.
+"
+tags: ["outreachy", "networkx", "first-time-contributor"]
+displayInList: true
+author: ["Paula Pérez Bianchi"]
+
+resources:
+- name: featuredImage
+  src: "outreachy.png"
+  params:
+    description: "Outreachy logo"
+    showOnTop: true
+---
+
+## What's Outreachy?
+
+**Outreachy** is a paid remote internship program for underrepresented groups in tech. All internships are in Open Source and Open Science. To be selected as an intern first you need to :
+
+1. _Fill out an initial application:_ You’ll need to answer some questions about how you are affected by the systemic bias, and how being underrepresented in your local tech industry impacted your development. Maybe you don’t know how to answer some of these questions, especially if you are still not looking for a job, but it’s important to do some research first. If you can’t find any “official” information, tech communities often do surveys and publish the results. Reaching out to local tech communities that work with underrepresented groups is a great way to find mentors and like-minded people that can support you through your tech career. Take your time to reflect on these questions before writing your answers. Also if you are a college student you need to submit your school calendar. Read carefully all the time requirements and reach out to Outreachy coordinators if you think there are any details about your school calendar that you need to discuss.
+
+2. _Take part in the contribution phase:_ Once your initial application is approved you will be able to see all the projects. Finding the right project for you is important and also very challenging. One can feel tempted to contribute to multiple projects but unless you have a lot of free time I don’t feel like it's the best option. Smaller and more constant contributions are the way to go. The contribution phase is not about introducing huge contributions but rather an opportunity to interact with the community, learn about the project, and gain new skills. Definitely finding the right project for you is key and depends a lot on how much time are you willing to put into it and your current skills.
+
+For more information about Outreachy go to: <https://www.outreachy.org/>
+
+## My experience at NetworkX during the contribution phase
+
+If this is your first time contributing to an open-source project you may feel overwhelmed. Understanding an almost 20-year-old project like NetworkX can feel like it's gonna take forever but don't worry I have some tips that may be handy for you during the contribution phase.
```

Comment:
```suggestion
If this is your first time contributing to an open-source project you may feel overwhelmed. Understanding an almost 20-year-old project like NetworkX can feel like it's going to take forever but don't worry I have some tips that may be handy for you during the contribution phase.
```

Just a suggestion to make the tone a little more formal - I don't want to put words in your mouth though, so please feel free to ignore!